packages/k10/generated-changes/overlay/questions.yml

```diff
@@ -0,0 +1,44 @@
+questions:
+### Storage Management
+
+- variable: global.persistence.storageClass
+  label: StorageClass Name
+  description: "Specifies StorageClass Name to be used for PVCs"
+  type: string
+  required: false
+  default: ""
+  group: "Storage Management"
+
+- variable: prometheus.server.persistentVolume.storageClass
+  type: string
+  label: StorageClass Name for prometheus PVC
+  description: "StorageClassName used to create Prometheus PVC. Setting this option overwrites global StorageClass value"
+  default: ""
+  required: false
+  group: "Storage Management"
+
+- variable: prometheus.server.persistentVolume.enabled
+  type: boolean
+  label: PVS for prometheus server
+  description: "If true, K10 Prometheus server will create a Persistent Volume Claim"
+  default: true
+  required: false
+  group: "Storage Management"
+
+- variable: global.persistence.enabled
+  type: boolean
+  label: Storage Enabled
+  description: "If true, K10 will use Perstistent Colume Claim"
+  default: true
+  required: false
+  group: "Storage Management"
+  
+### Service Account
+
+- variable: serviceAccount.name
+  description: "Name of a service account in the target namespace that has cluster-admin permissions. This is needed for the K10 to be able protect cluster resources."
+  type: string
+  label: Service Account Name
+  required: false
+  default: ""
+  group: "Service Account"
```

Comment:
the above file specifies the helm flag that we would be allowing users to customize while installing the chart.